packages/web/src/components/image-selection/ImageSelectionButton.js

```diff
@@ -84,6 +84,7 @@ const ImageSelectionButton = ({
             leftIcon={<IconPencil />}
             type={ButtonType.WHITE}
             onClick={handleClick}
+            buttonType='button'
```

Comment:
feels redundant 😅
should this be a default?

Reply:
right!?? so apparently the default is "submit" which is absolutely wild. if we want to change that in stems could be down